crypto/evp/digest.c

```diff
@@ -86,7 +85,6 @@ static int evp_md_ctx_reset_ex(EVP_MD_CTX *ctx, int keep_fetched)
         EVP_PKEY_CTX_free(ctx->pctx);
```

Comment:
Without this change we get a memory leak.. since the digestsign interface uses ctx->pctx

Reply:
Line 565 of self_test_kats.c..

Reply:
I think you meant that last comment to go against the other thread about the init field in st_kat_sign_st... :-)